shared/cmd/parser_test.go

```diff
@@ -0,0 +1,162 @@
+package cmd_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/mpvl/subtest"
```

Comment:
We can't use that. I seem to have also missed it getting into lxd/db/query/slices_test.go before. Can you please drop that from both places?

Reply:
I've vendored those two files in shared/lxd, since they are just two functions of respectively one (Go 1.7) and two lines (Go 1.6). Hope that's fine.